lib/src/main/java/com/auth0/android/lock/views/FormLayout.java

```diff
@@ -137,7 +137,7 @@ private void displayInitialScreen() {
     }
 
     private void addSocialLayout() {
-        socialLayout = new SocialView(lockWidget, true);
+        socialLayout = new SocialView(lockWidget, false);
```

Comment:
Param is no longer used, but the class is public and can't change the signature. Moved to "false" to be more explicit.